### PR TITLE
chore: clean up test data

### PR DESCRIPTION
**Pull request by chandumlg**
## What are the changes introduced in this PR?

clean up test data

## What is the related Linear task?

Resolves INT-1331

## Please explain the objectives of your changes below

Make test cases readable

### Any changes to existing capabilities/behaviour, mention the reason & what are the changes ?

N/A

### Any new dependencies introduced with this change?

N/A

### Any new generic utility introduced or modified. Please explain the changes.

N/A

### Any technical or performance related pointers to consider with the change?

N/A

<hr>

### Developer checklist

- [x] My code follows the style guidelines of this project

- [x] **No breaking changes are being introduced.**

- [ ] All related docs linked with the PR?

- [ ] All changes manually tested?

- [ ] Any documentation changes needed with this change?

- [ ] Is the PR limited to 10 file changes?

- [ ] Is the PR limited to one linear task?

- [ ] Are relevant unit and component test-cases added?

### Reviewer checklist

- [ ] Is the type of change in the PR title appropriate as per the changes?

- [ ] Verified that there are no credentials or confidential data exposed with the changes.

### CODE REVIEW

**Review by koladilip — APPROVED** [review]
good start to reuse the test data instead of repeating it multiple times.  cc: @utsabc

File: test/controllerUtility/ctrl-utility.test.ts
Comment (krishna2020) [inline test/controllerUtility/ctrl-utility.test.ts:1]: As this comes under unit test & unit tests co-exists within the code directory - can we move these tests to here https://github.com/rudderlabs/rudder-transformer/blob/a96e549b7f62bda8028e7cfe027eaa88243eb08d/src/controllers/util/index.test.ts ?

File: test/controllerUtility/ctrl-utility.test.ts [hunk 1]
```diff
@@ -916,520 +279,124 @@ const timestampEventsCases: timestampTestCases[] = [
...
-          Enabled: true,
-          WorkspaceID: 'wspId',
-          Transformations: [],
-        },
+        }),
+        metadata: metadata,
+        destination: destination,
       },
     ],
   },
```
Comment (krishna2020) [inline test/controllerUtility/ctrl-utility.test.ts:402]: Also this functionality is applicable only for rETL connection events, can update 
describe('controller utility tests -- handleTimestampInEvents `for retl connections`',

File: test/controllerUtility/ctrl-utility.test.ts
Comment (chandumlg) [reply test/controllerUtility/ctrl-utility.test.ts:1]: Moved.

**Review by krishna2020 — APPROVED** [review]

**Review by sanpj2292 — APPROVED** [review]

**Pull request merged**